### ISSUE TITLE
Reproduce the performan in CircleLoss?

### ISSUE THREAD

**Issue by GuHongyang** [KevinMusgrave/pytorch-metric-learning#419]
Hello, I would like to know if it is possible to reproduce the results of the CircleLoss paper using BN-Inception on the CUB-200-2011 dataset? What is the specific configuration? Thank you so much! !

**Comment by KevinMusgrave** [comment]
I took a quick look at the paper, and I believe this is their setting:

```python
from pytorch_metric_learning import losses, samplers

batch_size = 80
loss_fn = losses.CircleLoss(m=0.4, gamma=80)
# pass dataset labels into sampler
sampler = samplers.MPerClassSampler(labels, m=5, batch_size=batch_size)
# pass sampler into dataloader
```

**Comment by GuHongyang** [comment]
Yes! I'm new to this task, and I just want to use the bn-inception on CUB200 dataset. What's the common training config for bn-inception on CUB200 dataset with circleloss? Such as the optimizer type, the learning rate, the number of epochs, etc.. Can you give a specific config.

**Comment by KevinMusgrave** [comment]
Oh, I'm not sure if the Circle Loss paper goes into those details. 

The optimizer, lr, and number of epochs tends to vary from paper to paper. But to get an idea, you could check out section 4.2 of the [Proxy Anchor paper](Proxy Anchor Loss for Deep Metric Learning).

**Comment by GuHongyang** [comment]
OK, thanks a lot!
